db.go

```diff
@@ -67,6 +81,18 @@ const emptyReputationTableSQL = `
 TRUNCATE TABLE reputation;
 `
 
+const createViolationReputationWeightsTableSQL = `
+CREATE TABLE IF NOT EXISTS violation_reputation_weights (
+violation_type varchar(128) PRIMARY KEY NOT NULL,
+reputation int NOT NULL CHECK (reputation >= 0 AND reputation <= 100),
```

Comment:
OK https://github.com/mozilla-services/tigerblood/pull/19/commits/c02ea4446641e7850a32894fbccecd7c48bf5ec1 should do the right thing.